tests/www/apply/test_process.py

```diff
@@ -1682,13 +1690,20 @@ def _accept_view_post_data(self, job_application, post_data=None):
             "hired_job": job_description.pk,
         }
         # CertifiedCriteriaForm
-        birth_country = CountryFranceFactory()
-        birth_place = CommuneFactory()
         certified_criteria_default_fields = {
-            "birth_country": birth_country.pk,
-            "birth_place": birth_place.pk,
+            "birth_country": extra_post_data.get("birth_country", CountryFranceFactory().pk),
+            "birth_place": extra_post_data.get("birth_country", CommuneFactory().pk),
```

Comment:
```suggestion
            "birth_place": extra_post_data.get("birth_place", CommuneFactory().pk),
```

nit, mais deux appels à `setdefault` serait plus clair.